src/nyc_trees/apps/event/views.py

```diff
@@ -209,18 +209,62 @@ def start_event_map_print_job(request, event_slug):
 
 
 def event_check_in_page(request, event_slug):
-    # TODO: implement
-    return {}
+    event = get_object_or_404(Event, group=request.group, slug=event_slug)
+    rsvps = EventRegistration.objects.filter(event=event)
+    users = [(row.user, row.did_attend) for row in rsvps]
+    return {
+        'group': request.group,
+        'event': event,
+        'users': users
+    }
 
 
+@transaction.atomic
 def check_in_user_to_event(request, event_slug, username):
-    # TODO: implement
-    pass
+    did_attend = True
+    event = get_object_or_404(Event, group=request.group, slug=event_slug)
+    try:
+        rsvp = EventRegistration.objects.get(event=event,
+                                             user__username=username)
+        rsvp.did_attend = did_attend
+        rsvp.save()
+    except EventRegistration.DoesNotExist:
+        return HttpResponseForbidden()
+    return {
+        'group': request.group,
+        'event': event,
+        'user': rsvp.user,
+        'did_attend': did_attend
+    }
 
 
+@transaction.atomic
 def un_check_in_user_to_event(request, event_slug, username):
-    # TODO: implement
-    pass
+    did_attend = False
+    event = get_object_or_404(Event, group=request.group, slug=event_slug)
+    try:
+        rsvp = EventRegistration.objects.get(event=event,
+                                             user__username=username)
+        rsvp.did_attend = did_attend
+        rsvp.save()
+    except EventRegistration.DoesNotExist:
+        return HttpResponseForbidden()
+    return {
+        'group': request.group,
+        'event': event,
+        'user': rsvp.user,
+        'did_attend': did_attend
+    }
```

Comment:
I looks like the only difference between `check_in_user_to_event` and `un_check_in_user_to_event` is the value of `did_attend`.  Consider refactoring this like so:
```python
def check_in_user_to_event(request, event_slug, username):
   return _toggle_event_checkin(request, event_slug, username, True)

def check_in_user_to_event(request, event_slug, username):
   return _toggle_event_checkin(request, event_slug, username, False)

@transaction.atomic
def _toggle_event_checkin(request, event_slug, username, did_attend):
    event = get_object_or_404(Event, group=request.group, slug=event_slug)
    try:
        rsvp = EventRegistration.objects.get(event=event,
                                             user__username=username)
        rsvp.did_attend = did_attend
        rsvp.save()
    except EventRegistration.DoesNotExist:
        return HttpResponseForbidden()
    return {
        'group': request.group,
        'event': event,
        'user': rsvp.user,
        'did_attend': did_attend
    }
```